app/_locales/am/messages.json

```diff
@@ -0,0 +1,1887 @@
+    "message": "አሁን አዲሱን MetaMask UI እየተጠቀሙ ነው።"
+  },
+  "unapproved": {
+    "message": "ያልተፈቀደ"
+  },
+  "unavailable": {
+    "message": "አይገኝም"
+  },
+  "units": {
+    "message": "መለኪያዎች"
+  },
+  "unknown": {
+    "message": "ያልታወቀ"
+  },
+  "unknownNetwork": {
+    "message": "የማይታወቅ የግል አውታረ መረብ"
+  },
+  "unknownNetworkId": {
+    "message": "ያልታወቀ የአውታረ መረብ መለያ ቁጥር"
+  },
+  "unknownQrCode": {
+    "message": "ስህተት፡ ያን የ QR ኮድ መለየት አልቻልንም"
+  },
+  "unknownCameraErrorTitle": {
+    "message": "ኤጭ! የሆነ ችግር ተፈጥሯል..."
+  },
+  "unknownCameraError": {
+    "message": "ወደ ስልክዎ ለመድረስ ሲሞከር የተፈጠረ ስህተት አለ። እባክዎ እንደገና ይሞክሩ..."
+  },
+  "unlock": {
+    "message": "ክፈት"
+  },
+  "unlockMessage": {
+    "message": "ያልተማከለ ድር ይጠባበቃል"
+  },
+  "updatedWithDate": {
+    "message": "የዘመነ $1"
+  },
+  "uriErrorMsg": {
+    "message": "URIs አግባብነት ያለው የ HTTP/HTTPS ቅድመ ቅጥያ ይፈልጋል።"
+  },
+  "usaOnly": {
+    "message": "ዩኤስኤ ብቻ"
+  },
+  "usedByClients": {
+    "message": "በተለያዩ ደንበኞች ጥቅም ላይ የዋለ ነው"
+  },
+  "useOldUI": {
+    "message": "የቀድሞውን UI ይጠቀሙ"
+  },
+  "userName": {
+    "message": "የተጣቃሚ ስም"
+  },
+  "validFileImport": {
+    "message": "የሚያስመጡትን ትክክለኛ ፋይል መምረጥ አለብዎ፡"
+  },
+  "vaultCreated": {
+    "message": "ካዝና ተፈጥሯል"
+  },
+  "viewAccount": {
+    "message": "መለያን ይመልከቱ"
+  },
+  "viewinExplorer": {
+    "message": "በኤክስፕሎረር ተመልከት"
+  },
+  "viewContact": {
+    "message": "ዕውቂያን ይመልከቱ"
+  },
+  "viewOnCustomBlockExplorer": {
+    "message": "በ $1ይመልከቱ"
+  },
+  "viewOnEtherscan": {
+    "message": "በ Etherscan ላይ ይመልከቱ"
+  },
+  "viewNetworkInfo": {
+    "message": "የአውታረ መረበ መረጃን እይ"
+  },
+  "visitWebSite": {
+    "message": "ድረ ገጻችንን ይጎብኙ"
+  },
+  "walletSeed": {
+    "message": "የቋት ዘር"
+  },
+  "warning": {
+    "message": "ማስጠንቀቂያ"
+  },
+  "welcomeBack": {
+    "message": "እንኳን በደህና ተመለሱ"
+  },
+  "welcome": {
+    "message": "ወደ MetaMask እንኳን ደህና መጡ"
+  },
+  "whatsThis": {
+    "message": "ይህ ምንድን ነው?"
+  },
+  "writePhrase": {
+    "message": "ይህን ሐረግ በቁራጭ ወረቀት ላይ ይጻፉና ደህንነቱ በተጠበቀ ቦታ ያስቀምጡት። የበለጠ ደህንነት ካስፈለገዎ፣ በተለያዩ ቁርጥራጭ ወረቀቶች ላይ ይጻፉትና እያንዳንዳንቸውን በ2-3 የተለያዩ ቦታዎች ያስቀምጡት።"
+  },
+  "yesLetsTry": {
+    "message": "አዎ፣ እንሞክር"
+  },
+  "youNeedToAllowCameraAccess": {
+    "message": "ይህን ባህርይ ለመጠቀም የካሜራን መድረስ መፍቀድ አለብዎት።"
+  },
+  "yourSigRequested": {
+    "message": "ፊርማዎ እየተጠየቀ ነው"
+  },
+  "youSign": {
+    "message": "እየፈረሙ ነው"
+  },
+  "yourPrivateSeedPhrase": {
+    "message": "የግል ዘር ሐረግዎ"
+  },
+  "yourUniqueAccountImage": {
+    "message": "ልዩ የመለያዎ ገጽታ "
+  },
+  "yourUniqueAccountImageDescription1": {
+    "message": "ይህ ምስል በአዲሱ የመለያ ቁጥርዎ ለእርስዎ በፕሮግራማዊ አሰራር የተዘረገፈ ነው።"
+  },
+  "yourUniqueAccountImageDescription2": {
+    "message": "ግብይትን ማረጋገጥ ባስፈለገዎ ቁጥር ይህን ምስል ያያሉ።"
+  },
+  "yourUniqueAccountImageDescription3": {
+    "message": "MetaMask የዘር ሐረግዎን በጭራሽ አይጠይቅዎትም! "
+  },
+  "zeroGasPriceOnSpeedUpError": {
+    "message": "በስፒድ አፕ ላይ ዜሮ የነዳጅ ዋጋ"
+  },
+  "showIncomingTransactions": "Show Incoming Transactions",
```

Comment:
It looks like these new locales still have these English messages at the bottom

Reply:
thinking about this one, the pulled down strings come with English placeholders. I'll try to see if I can exclude.